docs/api/helpers/DirectionalLightHelper.html

```diff
@@ -63,7 +63,7 @@
 			light's [page:Object3D.matrixWorld matrixWorld].
 		</div>
 
-		<h3>[property:hex overrideColor]</h3>
+		<h3>[property:hex color]</h3>
 		<div>
 			A color that overrides the default color of the helper. If this was not passed in the constructor 
 			but is set later, the helper's color will update the next time [page:.update update] is called.
```

Comment:
I do not think all this description is necessary. Just say the color is optional and overrides the default color of the helper.